ecommerce/templates/ecommerce/product_detail.html

```diff
@@ -39,5 +40,95 @@ <h5 class="card-text font-weight-bold mr-4">{% trans 'Price : ' %}{{ object.pric
       </div>
     </div>
   </div>
+  <div class="col-lg-12 col-md-12">
+    <hr class="mt-4 mb-3">
+  </div>
+  <div class="row pt-4">
+    <div class="col-md-7">
+      <div class="d-flex justify-content-end pb-4">
+        <div class="d-flex align-items-center flex-nowrap">
+          <label class="fs-sm text-muted text-nowrap me-2 d-none d-sm-block" for="sort-reviews">Sort by: </label>
+          <div class="form-element form-select-sm" style="padding-left: 10px; padding-bottom: 10px;">
+            <select class="form-control" id="sort-reviews">
+              <option>Newest</option>
+              <option>Oldest</option>
+            </select>
+          </div>
+        </div>
+      </div>
+      <div class="product-review pb-4 mb-4 border-bottom">
+        <div class="d-flex mb-3">
+          <div class="d-flex align-items-center me-4 pe-2">
+            <img class ="rounded-circle" width="50" src="\media\profile_pics\avatar.png" alt="User">
+            <div class="ps-3" style="padding-left: 20px;">
+              <h6 class="fs-sm mb-0">Ngọc Duyên</h6>
+              <span class="fs-ms text-muted">June 28, 2019</span>
+            </div>
+            <div>
+              <div class="star-rating">
+                <i class="star-rating-icon ci-star-filled active"></i>
+                <i class="star-rating-icon ci-star-filled active"></i>
+                <i class="star-rating-icon ci-star-filled active"></i>
+                <i class="star-rating-icon ci-star-filled active"></i>
+                <i class="star-rating-icon ci-star"></i>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p class="fs-md mb-2">
+          Sản phẩm vip pro
+        </p>
+      </div>
+    </div>
+    
+    <!-- Reviview Form -->
+    <div class="col-md-5 mt-2 pt-4 mt-md-0 pt-md-0">
+      <div class="card p-4">
+        <h3 class="h4 pb-2">Write a review</h3>
+        <form method="post" class="row padding-top" action="">
+        <div class="h6 pb-2 col-sm-12">
+          <label class="form-label" for="review-name">Your name<span class="text-danger">*</span></label>
+          <input type="text" class="form-control" placeholder="Your name" required id="review-name">
+          <div class="invalid-feedback">Please enter your name!</div>
+          <small class="form-text text-muted">Will be displayed on the comment.</small>
+        </div>
+        <div class="h6 pb-2 col-sm-12">
+          <label class="form-label" for="review-email">Your email<span class="text-danger">*</span></label>
+          <input type="text" class="form-control" placeholder="Your email" required id="review-email">
+          <div class="invalid-feedback">Please provide valid email address!</div>
+          <small class="form-text text-muted">Authentication only - we won't spam you.</small>
```

Comment:
Chức năng review chỉ dành cho Khách hàng đã từng mua sản phẩm => Đã logged in, vậy thì đừng bắt nhập quá nhiều thông tin như này thì họ sẽ ngại và cảm thấy phiền toái khi review đó em nha. mặc định lấy thông tin theo user thôi